compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -251,6 +251,7 @@ OMR::X86::CodeGenerator::initialize(TR::Compilation *comp)
       {
       self()->setUseSSEForSinglePrecision();
       self()->setUseSSEForDoublePrecision();
+      self()->setSupportsAutoSIMD();
```

Comment:
I would like this to be guarded with an environment variable check - if the environment variable is not set then don't set this. The issue I have is that this is a cheap check for SIMD support that saves compile-time. By saying yes we support AutoSIMD but then saying no to every operation we increase compile-time without providing any benefit by applying SIMD to something. As such until the codegen can actually support some kind of SIMD I'd like for this to remain disabled by default to avoid performance regressions and compile-time changes that have no possible benefit.